src/Tools/dotnet-trace/build.cmd

```diff
@@ -1,7 +1,7 @@
 @if not defined _echo echo off
 cls
 
-dotnet.exe restore "%~dp0dotnet-trace.csproj" --packages "%~dp0..\..\..\artifacts\packages" || (
```

Comment:
Should we even have this script? It isn't building it with arcade.

/cc: @jorive

Reply:
I would prefer to keep it because:
- It does not use Arcade,
- It does not restore to the global cache,
- It builds the single project I need,
- `Build.cmd` does not always build successfully "Any CPU" because the native dependencies,
  - Maybe this can be fixed by adding a "Mixed" mode to build native and managed projects?

This is the workflow I follow when working on `dotnet-trace` related tooling, but I understand that it seems weird to have a script here and not in the other projects.

Reply:
This builds with arcade targets/props as expected. Sure, it doesn't run testing, signing, and all of that, but this is a dev script. I like it in some was because it doesn't always rely on polluting the global cache (that being said NuGet *will* use the cache as a first target to obtain the packages anyway).